### DISCUSSION TITLE
show/hidde text

### DISCUSSION THREAD

**Discussion by cylace** [tailwindlabs/tailwindcss#12372] [Help] [answered]
Hi everybody,
i've a component checkbox (with two input yes and no) and a parent vue. Thus i will have a div bloc on parent vue who is showed if user click on yes checkbox and hidde if user click on no checkbox. I tried with peer and peer-checked but don't work. If i try this on the same page that's right but with two that don't work. Thank you for your help.

**Comment by alex-krasikau** [comment]
Hey @cylace!

I created an example of how to do it using peer class. Please check it out - https://play.tailwindcss.com/IyW5TWAqxN

**Comment by cylace** [comment]
Hey thank you, problem is that i've my two input on my component and i use that component on an parent view and i must show/hidde that block on parent view.

**Reply by alex-krasikau** [reply]
In this situation, the use of `:has` may be helpful. However, Tailwind CSS does not currently support it because `:has` lacks full support across all major browsers. If it's feasible for you, there's a solution - https://play.tailwindcss.com/nStXJiDR8G.

You can visit https://caniuse.com/css-has to check which browsers support this feature.

**Comment by cylace** [comment]
hum don't work for my code. I will maybe search with other method (i use for that vue.js with inertia). Thank you for your help.

**Reply by alex-krasikau** [reply, marked as answer]
It would be much easier to handle it via state in Vue
